.eslintrc

```diff
@@ -0,0 +1,94 @@
+{
+    'parser': 'babel-eslint',
+    'extends': [
+        'airbnb-base',
+    ],
+    'env': {
+        'es6':     true,
+        'browser': true,
+        'amd':     true,
+        'mocha':   true,
+        'jquery':  true,
+    },
+    'globals': {
+        'jQuery':  false,
+        '$':       false,
+        'require': false,
+        'module':  false,
+        'console': false,
+        'global':  false,
+
+        'dataLayer':       true,
+
+        'page':            false,
+        'TUser':           false,
+        'BinarySocket':    false,
+        'LocalStore':      false,
+        'SessionStore':    false,
+        'load_with_pjax':  false,
+        'Highcharts':      false,
+        'GTM':             false,
+        'trackJs':         false,
+        'DESK':            false,
+        'getClockStarted': false,
+        'MBPrice':         false,
+
+        'processActiveSymbols':       false,
+        'processMarket':              false,
+        'processContract':            false,
+        'processContractForm':        false,
+        'forgetTradingStreams':       false,
+        'processForgetProposals':     false,
+        'processPriceRequest':        false,
+        'processForgetTicks':         false,
+        'processTick':                false,
+        'processProposal':            false,
+        'processTradingTimesRequest': false,
+        'processTradingTimes':        false,
+
+        'processActiveSymbols_Beta':       false,
+        'processMarket_Beta':              false,
+        'processContract_Beta':            false,
+        'processContractForm_Beta':        false,
+        'forgetTradingStreams_Beta':       false,
+        'processForgetProposals_Beta':     false,
+        'processPriceRequest_Beta':        false,
+        'processForgetTicks_Beta':         false,
+        'processTick_Beta':                false,
+        'processProposal_Beta':            false,
+        'processTradingTimesRequest_Beta': false,
+        'processTradingTimes_Beta':        false,
+    },
+    'rules': {
+        'array-callback-return': 0,
+        'func-names':  ['error', 'never'],
+        'indent':      ['error', 4],
+        'key-spacing': ['error', { align: 'colon' }],
```

Comment:
don't think it's a good idea to align colons. In my opinion, such code looks nasty